docs/application/native/get-started/iot/first-app.md

```diff
@@ -0,0 +1,292 @@
+# Create Your First Tizen IoT Native Service Application
+
+**Welcome to Tizen IoT Native Service Application development!**
+
+> [!NOTE]
+> Service applications are Tizen native applications which run in the background with no graphical user interface.
+> They can be very useful for activities, such as getting sensor data in the background, that need to be run periodically or continuously but do not require any user interaction.
+
+An IoT native service application can be created using the C language and run on the Raspberry Pi. The application uses the native APIs, which provide various interfaces to the device hardware allowing you to take advantage of numerous capabilities tailored to run with limited device resources.
```

Comment:
Global Question: Would it be good to specify the Raspberry Pi version supported? For example Raspberry Pi 3 or 4
==> Since Tizen 5.0 to later versions support Raspberry Pi 3. And since Tizen 6.0 to later version supports Raspberry Pi 4. 
I think I can't write "3 or 4" because the version of Raspberry Pi supported by Tizen version is different. So now it is written as Raspberry Pi comprehensively.
We will make a page to explain the version of Tizen and Raspberry Pi later. Once the page is created, how about putting it on the link?

Suggest removing article "the" before Raspberry Pi:
==> Thank you for your suggestion. I will remove "the" before Raspberry Pi.

Reply:
Thank you for the exaplanation. yes, we can add the link once the page is ready.